doc/languages-frameworks/python.section.md

```diff
@@ -1482,20 +1482,21 @@ If you need to change a package's attribute(s) from `configuration.nix` you coul
   nixpkgs.config.packageOverrides = super: {
     python = super.python.override {
       packageOverrides = python-self: python-super: {
-        zerobin = python-super.zerobin.overrideAttrs (oldAttrs: {
-          src = super.fetchgit {
-            url = "https://github.com/sametmax/0bin";
-            rev = "a344dbb18fe7a855d0742b9a1cede7ce423b34ec";
-            sha256 = "16d769kmnrpbdr0ph0whyf4yff5df6zi4kmwx7sz1d3r6c8p6xji";
+        twisted = python-super.twisted.overrideAttrs (oldAttrs: {
+          src = super.fetchPipy {
+            pname = "twisted";
+            version = "19.10.0";
+            sha256 = "7394ba7f272ae722a74f3d969dcf599bc4ef093bc392038748a490f1724a515d";
+            extension = "tar.bz2";
           };
         });
       };
     };
   };
 ```
 
-`pythonPackages.zerobin` is now globally overridden. All packages and also the
-`zerobin` NixOS service use the new definition. Note that `python-super` refers
+`pythonPackages.twisted` is now globally overridden. All packages and also the
+`graphite` NixOS service use the new definition. Note that `python-super` refers
```

Comment:
Feel free to take or leave this suggestion: the reader might need to connect the dots that `graphite` uses `twisted` - so maybe

```suggestion
`pythonPackages.twisted` is now globally overridden. All packages and also the
`graphite` NixOS service, which uses `twisted`, use the new definition. Note that `python-super` refers
```

or 

```suggestion
`pythonPackages.twisted` is now globally overridden. All packages and also all
NixOS services that reference `twisted` use the new definition. Note that `python-super` refers
```

Reply:
Thanks, I've merged your second suggestion.